### PR TITLE
Fix parsing of IBM vpc hostname

### PR DESCRIPTION
**Pull request by k-chu**
Fixes https://github.com/ManageIQ/manageiq-providers-ibm_terraform/issues/52

### CODE REVIEW

**Comment by k-chu** [comment]
@miq-bot add_label bug

File: app/models/manageiq/providers/ibm_terraform/inventory/parser/configuration_manager.rb [hunk 1]
```diff
@@ -88,7 +88,7 @@ def get_hostname(virtual_machine)
     if vm_provider == "Amazon EC2"
       virtual_machine.dig("details", "tags.Name") || virtual_machine.dig("details", "tags", "Name")
     elsif vm_provider == "IBM"
-      virtual_machine.dig("details", "hostname")
+      virtual_machine.dig("details", "hostname") || virtual_machine.dig("details", "name")
```
Comment (agrare) [inline app/models/manageiq/providers/ibm_terraform/inventory/parser/configuration_manager.rb:91]: Can the hostname actually be in either of these places or you're just using the name as the hostname so it won't be `nil`?

Comment (k-chu) [reply app/models/manageiq/providers/ibm_terraform/inventory/parser/configuration_manager.rb:91]: @agrare The hostnames are located in different field depending on which IBM Cloud infrastructure the vm was deployed to. (classic infrastructure(aka old SoftLayer) vs VPC)

Comment (agrare) [reply app/models/manageiq/providers/ibm_terraform/inventory/parser/configuration_manager.rb:91]: :+1:

Comment (agrare) [reply app/models/manageiq/providers/ibm_terraform/inventory/parser/configuration_manager.rb:91]: @k-chu can you add that to the PR description?

**Review by agrare — APPROVED** [review]

**Comment by simaishi** [comment]
Kasparov backport details:

```
$ git log -1
commit f63004293aa9dbdd9a7d334fb9d237f0e0192253
Author: Adam Grare <agrare@redhat.com>
Date:   Mon Dec 7 14:50:34 2020 -0500

    Merge pull request #53 from k-chu/fix_ibm_vps_hostname

    Fix parsing of IBM vpc hostname

    (cherry picked from commit f762fd0049291ae3c5966130c6272f74e381195f)
```

**Pull request merged**